patches/server/0879-Implement-getComputedBiome-API.patch

```diff
@@ -0,0 +1,66 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jason Penilla <11360596+jpenilla@users.noreply.github.com>
+Date: Mon, 14 Mar 2022 22:46:05 -0700
+Subject: [PATCH] Implement getComputedBiome API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftRegionAccessor.java b/src/main/java/org/bukkit/craftbukkit/CraftRegionAccessor.java
+index 99cb7625d50d5da4ce0999e10fb84403958a7ffe..2a9289758493832537a9cea0bbcb3bbe7e062691 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftRegionAccessor.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftRegionAccessor.java
+@@ -204,6 +204,18 @@ public abstract class CraftRegionAccessor implements RegionAccessor {
+         return CraftBlock.biomeBaseToBiome(this.getHandle().registryAccess().registryOrThrow(Registry.BIOME_REGISTRY), this.getHandle().getNoiseBiome(x >> 2, y >> 2, z >> 2));
+     }
+ 
++    // Paper start
++    @Override
++    public Biome getComputedBiome(Location location) {
```

Comment:
ugh.... location really doesn't work well here. maybe a precondition to make sure its the same world? kinda odd cause this can be run in a world-gen context.